Gemfile

```diff
@@ -5,8 +5,14 @@ gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
 gem 'pg', '~> 0.18'
 gem 'puma', '~> 3.0'
 gem 'jbuilder', '~> 2.5'
+gem 'rack-cors', require: 'rack/cors'
 
 group :development, :test do
+ gem 'rspec-rails'
+ gem 'shoulda-matchers'
+ gem 'factory_girl_rails'
```

Comment:
I think it is best to replace factory_girl_rails with factory_bot_rails as it is the latest version of the gem.